plugin/bbb/cron_close_meeting.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * This script initiates a video conference session, calling the BigBlueButton API
+ * @package chamilo.plugin.bigbluebutton
+ */
+
+$course_plugin = 'bbb'; //needed in order to load the plugin lang variables
+require_once __DIR__.'/config.php';
+
+$plugin = BBBPlugin::create();
+$meetingTable = Database::get_main_table('plugin_bbb_meeting');
+$roomTable = Database::get_main_table('plugin_bbb_room');
+
+$bbb = new bbb();
+if ($bbb->pluginEnabled) {
+    $activeSessions = $bbb->getActiveSessions();
+
+    if (!empty($activeSessions)) {
+        foreach ($activeSessions as $value) {
+            $meetingId = $value['id'];
+
+            $courseInfo = api_get_course_info_by_id($value['c_id']);
+            $courseCode = $courseInfo['code'];
+
+            $meetingBBB = $bbb->getMeetingInfo(
+                [
+                    'meetingId' => $value['remote_id'],
+                    'password' => $value['moderator_pw']
+                ]
+            );
+
+            if ($meetingBBB === false) {
+                //checking with the remote_id didn't work, so just in case and
+                // to provide backwards support, check with the id
+                $params = array(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Short array syntax must be used to define arrays](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5492268)

Reply:
Corregir los errores de sintaxis